testing/spectest/mainnet/capella/epoch_processing/BUILD.bazel

```
@@ -0,0 +1,25 @@
+load("@prysm//tools/go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    size = "small",
+    srcs = [
+        "effective_balance_updates_test.go",
+        "eth1_data_reset_test.go",
+        "historical_roots_update_test.go",
+        "inactivity_updates_test.go",
+        "justification_and_finalization_test.go",
+        "participation_flag_updates_test.go",
+        "randao_mixes_reset_test.go",
+        "registry_updates_test.go",
+        "rewards_and_penalties_test.go",
+        "slashings_reset_test.go",
+        "slashings_test.go",
+    ],
+    data = glob(["*.yaml"]) + [
```

```suggestion
    data = [
```
There are no yaml files here that would match that glob.